frontend/src/components/Members/GMemberDialog.vue

```diff
@@ -240,6 +240,7 @@ export default {
     rules.internalRoles = withFieldName('Member Roles', internalRolesRules)
 
     if (this.isUpdateDialog) {
+      rules.internalName = {}
```

Comment:
instead, initialize it already here

```js
const rules = {
  internalName: {}
}
```

Reply:
I still do not understand why not just remove the `if (this.isUpdateDialog) {`

Reply:
Added comment to make it clear